compiler/luci/pass/src/FuseBatchNormWithTConvPass.cpp

```diff
@@ -215,6 +218,7 @@ bool fused_batch_norm_with_tconv(luci::CircleAdd *add)
   fused_tconv->stride()->h(tconv->stride()->h());
   fused_tconv->stride()->w(tconv->stride()->w());
   fused_tconv->name(name + "/TransposeConv");
+  fused_tconv->fusedActivationFunction(tconv->fusedActivationFunction());
```

Comment:
```suggestion
  // TODO Remove following ReLU Ops and set proper fused activation function
  fused_tconv->fusedActivationFunction(luci::FusedActFunc::NONE);
```

Reply:
`tconv->fusedActivationFunction()` is always NONE, so there will be no problem, but logically fused tconv should not have the same activation function with tconv.

Reply:
It's not always NONE. One of our internal model is RELU with new TF2.12.1, which I first started upgrading the schema version.

Reply:
Thanks!, I'll update with NOTE.